### ISSUE TITLE
LR Schedule

### ISSUE THREAD

**Issue by sondo2503** [zylo117/Yet-Another-EfficientDet-Pytorch#583]
Thanks for your implementation. I have a question about your LR schedule. Why you call scheduler.step() for your train_loss instead of val_loss?

**Comment by zylo117** [comment]
val set is used to validate, it should not interfere the training.
